active/ImageIcon/ImageIconSpec.md

```diff
@@ -0,0 +1,123 @@
+
+ImageIcon is a superset of the existing BitmapIcon minus the
+[BitmapIcon.ShowAsMonochrome](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIcon.ShowAsMonochrome)
+property, as it is not a requirement from design to have all icons be monochrome.
+Since ShowAsMonochrome will not be part of ImageIcon, the 
+[IconElement.Foreground](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconElement.Foreground)
+inherited from IconElement will not be suported in this control.
+It will be up to the application to update the icon to match the user's theme. 
+
+Along with the various IconElement types, which are elements, there are IconSource types.
+For example BitmapIcon and
+[BitmapIconSource](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.BitmapIconSource).
+Similarly in this spec, along with `ImageIcon` is an `ImageIconSource`.
+
+# API Pages
+
+## ImageIcon class
+
+Represents an icon that uses an image as its content.
+
+ImageIcon can be used just like an
+[Image](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Image)
+element, except it derives from
+[IconElement](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.IconElement)
+and can therefore be used anywhere an IconElement is required, such as
+[NavigationViewItem.Icon](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.NavigationViewItem.Icon).
+
+> Note: the ImageIcon.Foreground property is currently ignored.
+
+## Is this the right control? 
+
+Use an ImageIcon control when you want to display an image as your icon. Do not use an ImageIcon if you want to display an icon from a font. 
+
+## Examples
+Below is an example of how you can use ImageIcon in an App Bar Button Item.
+
+> Spec note: This example will also be added to the App Bar Button Item page with the rest of the icon controls. 
+
+XAML
+```xml
+<AppBarButton Label="ImageIcon" Click="AppBarButton_Click">
+    <AppBarButton.Icon>
+        <ImageIcon Source="Wheel.png"/>
+    </AppBarButton.Icon>
+</AppBarButton>
+```
+
+## ImageIcon member notes
+
+| Name	| Type | Description | Default |
+|:--- | :--- | :--- | :--- |
+|Source | ImageSource | Gets and sets the icon source file that will be used in the application. | Null |
+
+# API Details
+
+```cs
+namespace Microsoft.UI.Xaml.Controls
+{
+    [webhosthidden]
+    unsealed runtimeclass ImageIcon : Windows.UI.Xaml.Controls.IconElement
+    {
+        ImageIcon();
+
+        Windows.UI.Xaml.Media.ImageSource Source{ get; set; };
+
+        static Windows.UI.Xaml.DependencyProperty SourceProperty{ get; };
+    }
+
+    [webhosthidden]
+    unsealed runtimeclass ImageIconSource : IconSource
+    {
+        ImageIconSource();
+
+        Windows.UI.Xaml.Media.ImageSource ImageSource{ get; set; };
+
+        static Windows.UI.Xaml.DependencyProperty ImageSourceProperty{ get; };
+    }
+}
+```
+
+# Appendix
+
+## Scope
+
+|Capability	| Priority |
+| Icon can be used anywhere a BitmapIcon is accepted today | Must|
```

Comment:
Nit: This table formatting isn't quite right
![image](https://user-images.githubusercontent.com/20007011/106316084-09a23400-6221-11eb-8ab7-9ed19f8405d8.png)